tox.ini

```diff
@@ -0,0 +1,31 @@
+[flake8]
+max-line-length = 100
+
+[tox]
+skipsdist = True
+
+[testenv]
+setenv =
+    PYTHONPATH={toxinidir}/src:{toxinidir}/lib
+deps =
+    -rrequirements.txt
+    -r test-requirements.txt
+
+[testenv:lint]
+deps =
+    flake8
+    black==20.8b1
+commands =
+    flake8 {toxinidir}/src {toxinidir}/tests
+    black --check {toxinidir}/src {toxinidir}/tests
+
+[testenv:unit]
+commands =
+    pytest -vv --tb native --show-capture=no --log-cli-level=INFO {toxinidir}/tests/unit
```

Comment:
I have no strong opinion and just copied it from somewhere else, but agree logs are usually good.  @DomFleischmann do you have an opinion?

Sidenote: we should make a template/style guide repo for our team that has these sorts of things in it.  Where we've got 4 people and 20 charms, I think we're often having the same debates (and probably, by accident, ignoring the decisions just because we forget)

Reply:
We spoke briefly yesterday about having a Github Template repo with all this laid out. I'll take a look and send over for a look

Reply:
removed the `--tb native` for now but parked the discussion in canonical/operator-template#4